src/components/sidebar/actions/productSlice.ts

```diff
@@ -68,8 +69,14 @@ export const productSlice = createSlice({
 
       const newGeneratedProducts: GeneratedProduct[] = productsToBeGeneratedCopy.map((granuleId, index) => {
         const relevantAddedProduct = state.addedProducts.find(productObj => productObj.granuleId === granuleId) as allProductParameters
-
         const {utmZoneAdjust, mgrsBandAdjust, cycle, pass, scene} = relevantAddedProduct
+        const {outputGranuleExtentFlag, outputSamplingGridType, rasterResolution} = state.generateProductParameters
+        const fetchData = async () => {
+          const userInfoResponse = await generateL2RasterProduct(cycle, pass, scene, outputGranuleExtentFlag, outputSamplingGridType, rasterResolution, utmZoneAdjust, mgrsBandAdjust)
```

Comment:
Is `userInfoResponse` the right name for that variable?

Reply:
i removed the variable name since it was not necessary. It is incorrectly named